pkg/minikube/problem/err_map.go

```diff
@@ -30,6 +30,11 @@ var vmProblems = map[string]match{
 		Advice: "Please make sure the service you are looking for is deployed or is in the correct namespace.",
 		Issues: []int{4599},
 	},
+	"MACHINE_NOT_FOUND": {
+		Regexp: re(`Machine does not exist for api.Exists(lis)`),
```

Comment:
the (lis) is extra ( that was my example profile)
it should be "Machine does not exist for api.Exists"

I recommend attaching the output of the console to try, I recmmend
deleting your minikube then 
then try ./out/minikube ssh -p p1 -- ls -lah

then paste the output in the PR review so we know it worked :)

Reply:
sure

Reply:
Done!